utils/monitoring.js

```diff
@@ -0,0 +1,113 @@
+const { errors } = require('arsenal');
+const promClient = require('prom-client');
+const { Registry } = require('prom-client');
+const CountMaster = require('../CountItems/CountMaster');
+
+const aggregatorRegistry = new promClient.AggregatorRegistry();
+const { collectDefaultMetrics } = promClient;
+
+const bucketCount = (promClient.register.getSingleMetric('bucketCount_metrics'))
+    || new promClient.Gauge({
+        name: 'bucketCount_metrics',
+        help: 'Total number of buckets',
+        labelNames: ['bucketName'],
+    });
+
+// Histogram of the bucket processing duration, by the utilization service.
+
+const bucketProcessingDuration = (promClient.register.getSingleMetric('count_items_bucketProcessingDuration'))
+    || new promClient.Histogram({
+        name: 'count_items_bucketProcessingDuration',
+        help: 'Duration of processing a bucket',
+        labelNames: ['service'],
+        buckets: [1, 10, 60, 600, 3600, 18000, 36000],
+    });
+
+const consolidationDuration = (promClient.register.getSingleMetric('count_items_consolidationDuration'))
+    || new promClient.Histogram({
+        name: 'count_items_consolidationDuration',
+        help: 'Duration of metrics consolidation in seconds',
+        labelNames: ['service'],
+        buckets: [0.1, 0.5, 1, 2, 5, 10],
```

Comment:
I think we can have smaller values, as no I/O is involved here

Reply:
changed here : https://github.com/scality/s3utils/pull/312/commits/845c8698405f12a5273fce139241140e7c4d42df#diff-612a4c10e405510cdaad879589d4a7d157df6e0ff42b4645d2a21d4d17e41fa9R31